### PR TITLE
Streamline the README

### PR DESCRIPTION
**Pull request by apurvis**
trying to make this as minimal an example as possible and lead people to the wiki for more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,68 +11,55 @@ Applications using broadside employ a configuration file that looks something li
 
 ```ruby
 Broadside.configure do |config|
-  config.application = 'hello_world'
-  config.docker_image = 'lumoslabs/hello_world'
-  config.ecs.cluster = 'production-cluster'
+  config.application =  'hello_world'
```
Comment (matl33t) [inline README.md]: i think it'd look nicer if the `=` were aligned

File: README.md [hunk 2]
```diff
@@ -11,68 +11,55 @@ Applications using broadside employ a configuration file that looks something li
...
+  config.docker_image = 'docker.io/lumoslabs/hello_world'
```
Comment (matl33t) [inline README.md]: is this new? images on dockerhub shouldn't need the domain.

File: README.md [hunk 3]
```diff
@@ -11,68 +11,55 @@ Applications using broadside employ a configuration file that looks something li
...
       ]
     },
     staging_web: {
-      cluster: 'staging-cluster', # Overrides config.ecs.cluster for this target only
       scale: 1,
       command: ['bundle', 'exec', 'puma'],
-      env_file: '../.env.staging',
-      tag: 'latest_staging' # Either configure the tag per target or pass --tag to the deploy command
+      env_file: '.env.staging',
+      tag: 'latest_staging',
```
Comment (matl33t) [inline README.md]: i think `latest` is fine

File: README.md [hunk 4]
```diff
@@ -12,67 +12,54 @@ Applications using broadside employ a configuration file that looks something li
 ```ruby
 Broadside.configure do |config|
   config.application = 'hello_world'
-  config.docker_image = 'lumoslabs/hello_world'
+  config.docker_image = 'docker.io/lumoslabs/hello_world'
```
Comment (apurvis) [inline README.md]: re: 
> is this new? images on dockerhub shouldn't need the domain.

i just made up some shit that looks like a quay URL to make it clear what was expected here; do you have a better suggestion?  i don't really know how that stuff works

Comment (matl33t) [inline README.md]: without a domain in the name docker will assume the image is stored in dockerhub, but we're using quay so specifying that domain before the project is necessary

**Pull request merged**